code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -29,9 +29,21 @@
 	bodytemp_heat_damage_limit = (BODYTEMP_HEAT_DAMAGE_LIMIT + 20) // This puts lizards 10 above lavaland max heat for ash lizards.
 	bodytemp_cold_damage_limit = (BODYTEMP_COLD_DAMAGE_LIMIT - 10)
 
-/// Lizards are cold blooded and do not stabilize body temperature naturally
+/// Lizards are cold blooded and do not stabilize body temperature naturally unless very cold

```

Comment:
Comment says it only works when very cold, but it seems you stabilize it when very hot too

Reply:
Updated the comment